luatest/hooks.lua

```diff
@@ -43,6 +67,11 @@ local function define_hooks(object, hooks_type)
             end
         end
     end
+    if preloaded_hook then
+        for _, v in pairs(preloaded_hook) do
```

Comment:
`pairs` does not guarantee any order. Use `ipairs` to traverse array-like tables in the order of indices.

Reply:
yep, fixed. Thanks!